kaggle_environments/main.py

```diff
@@ -257,15 +234,8 @@ def http_request(request, middleware):
             del params[key]
 
     body = request.get_json(silent=True, force=True) or {}
-
     req = parse_args({**params, **body})
-    if middleware["request"] is not None:
```

Comment:
middleware feature is no longer used with the move to the new evaluation system.